client/render/gl_grass.cpp

```diff
@@ -1112,7 +1112,7 @@ find or add unique texture for grass
 byte R_GrassTextureForName( const char *name )
 {
 	byte i;
-	for( i = 0; i < GRASS_TEXTURES && grasstexs[i].name[0]; i++ )
```

Comment:
What's the point of this change?

Reply:
There infinite loop is possible.

Reply:
It's better to not use such small types like byte for iteration. It gives no advantage, just use int or size_t.